client/relay-chain-minimal-node/src/lib.rs

```diff
@@ -200,24 +202,47 @@ async fn new_minimal_relay_chain(
 	Ok(NewMinimalNode { task_manager, overseer_handle, network })
 }
 
+/// Usually availability-distribution subsystem handles these requests.
+/// We do not run it in this minimal node and therefore we answer
+/// that we do not have the chunk/PoV.
+fn drain_unwanted_request_channels(
+	task_manager: &TaskManager,
+	mut chunk_req_receiver: IncomingRequestReceiver<v1::ChunkFetchingRequest>,
+) {
+	task_manager.spawn_handle().spawn("chunk-request-drainer", None, async move {
+		loop {
+			if let Ok(request) = chunk_req_receiver.recv(|| vec![COST_INVALID_REQUEST]).await {
+				tracing::debug!(
+					target: LOG_TARGET,
+					"Received chunk fetching request. Collators should not receive this."
```

Comment:
Is it a protocol violation to ask chunks/PoVs from collator nodes?

Reply:
The [spec](https://spec.polkadot.network/#sect-candidate-recovery) says you should request this from validators. Everything else also does not make sense. But you can still send the request to everyone else running the availability-distribution subsystem, and they will [respond with `NoSuchChunk`](https://github.com/paritytech/polkadot/blob/e1cd7cec7a7bf4e441ff500fa4137c1cb526dd1e/node/network/availability-distribution/src/responder.rs).